### ISSUE TITLE
Is it possible to rewrite media url from other cdn?

### ISSUE THREAD

**Issue by johnsyin** [humanmade/S3-Uploads#556]
I believe this is more like a feature request. Will close the issue if it cannot be easily done.

I am an S3-Uploads user for a while now and now I am thinking to reduce a bit of cost to move to a free CDN (cloudflare), by manually uploading my media.

Is it possible to use the same mechanism (linking to external URL) for all media files like S3-Uploads? As I notice the URL is not modifiable in each media in the wp admin panel.

**Comment by pc-erin** [comment]
Yes, the `S3_UPLOADS_BUCKET_URL` define should be set to the root path of your CDN pull zone.

In the case of cloudflare though, I don't think the path matters. You put the entire site behind cloudflare and it automatically serves your media files from local CDN endpoints.
